packages/airnode-admin/src/cli.ts

```diff
@@ -443,6 +443,14 @@ yargs
       console.log(`Is requester whitelisted: ${isRequesterWhitelisted}`);
     }
   )
+  .command(
+    'generate-mnemonic',
+    'Generates a random mnemonic. Uses "ethers.Wallet.createRandom" under the hood.',
+    async () => {
+      const mnemonic = await admin.generateMnemonic();
+      console.log(`Generated mnemonic: ${mnemonic}`);
```

Comment:
Do we want to add some warnings here about writing it down somewhere safe?

Reply:
I do not think it's necessary. The user have an intention to generate the mnemonic for. He should know where to write the mnemonic and how to store it (or write it down).